comp/api/api/apiimpl/internal/config/endpoint.go

```diff
@@ -94,13 +99,28 @@ func (c *configEndpoint) getConfigValueAsJSON(r *http.Request) ([]byte, int, err
 	vars := gorilla.Vars(r)
 	path := vars["path"]
 
-	if _, ok := c.authorizedConfigPaths[path]; !ok {
+	authorized := false
+	if _, ok := c.authorizedConfigPaths[path]; ok {
+		authorized = true
+	} else {
+		// check to see if we have a prefix rule path (trailing period) which matches as the prefix
+		// for the given path
```

Comment:
Same question than https://github.com/DataDog/datadog-agent/pull/22245#discussion_r1466198176: Since this is an HTTP API, should we not query `ha` setting like `/config/ha` for everything and `/config/ha/site` for `ha.site` ?

Reply:
Same answer as Pierre. 😅 

I think it makes the most sense (even if it's not the most intuitive for unfamiliar users) to stick with the key format that we use when querying the configuration internally.